skimage/measure/_regionprops.py

```diff
@@ -166,6 +167,31 @@ def extent(self):
         return self.moments[0, 0] / (rows * cols)
 
     @property
+    def feret_diameter(self):
+        from ..morphology.convex_hull import convex_hull_image
+        from ._find_contours import find_contours
+        
+        label_image = self._label_image
+        label = self.label
+        
+        max_feret = 0.0
+        identity_convex_hull = convex_hull_image(label_image == label)
+        coordinates = np.vstack(find_contours(identity_convex_hull, 0.5, fully_connected = 'high'))
+        
+        n = len(coordinates)
+        for a in range(0, n):
+            for b in range(0, n):
+                dx = coordinates[a][0] - coordinates[b][0]
+                dy = coordinates[a][1] - coordinates[b][1]
+                d = sqrt(dx ** 2 + dy ** 2)
+                if d > max_feret:
+                    max_feret = d
+                    i1 = a
```

Comment:
I originally included the points used to compute the distance as part of a returned tuple, but I ran into some static when the test harness tried to convert a (float, (float, float), (float, float)) into an array. Would it be better to have a return tuple such as (distance, x0, y0, x1, y1)?